### DISCUSSION TITLE
icons with home assistant 2024.02

### DISCUSSION THREAD

**Discussion by thejob78** [marcokreeft87/room-card#315] [General]
With the updates to Home Assistant 2024.02,
many icons in custom:room-card tabs have become incorrect, it seems to show a default icon
for example those of the covers and the media
![image](https://github.com/marcokreeft87/room-card/assets/17272116/fd2928c7-7d2b-4514-a5d1-f343a4e49bef)

**Comment by marcokreeft87** [comment]
Duplicate of https://github.com/marcokreeft87/room-card/issues/313
